virttest/utils_misc.py

```diff
@@ -3820,16 +3820,18 @@ class SELinuxBoolean(object):
 
     def __init__(self, params):
         self.server_ip = params.get("server_ip", None)
+        self.ssh_user = params.get("server_user", "root")
+        self.params = params
         if self.server_ip:
             # Setup SSH connection
             from utils_conn import SSHConnection
             self.ssh_obj = SSHConnection(params)
             ssh_timeout = int(params.get("ssh_timeout", 10))
             self.ssh_obj.conn_setup(timeout=ssh_timeout)
+            self.check_if_ubuntu_remote = self.check_if_ubuntu(remote=True)
```

Comment:
Instead hard code a new function for ubuntu in this Class, I think you may need a more general way to check if 'selinux' is supported in the place where calling the config function.